Sim/SimG4Components/src/G4SimAlg.cpp

```diff
@@ -43,7 +45,17 @@ StatusCode G4SimAlg::initialize() {
 
 StatusCode G4SimAlg::execute() {
   // first translate the event
-  G4Event* event = EDM2G4();
```

Comment:
Since you moved the translation into the Action, the G4SimAlg does not need to `declareInput` anymore.